src/store/modules/ProfileModule.ts

```diff
@@ -14,7 +14,6 @@ import ManagerSettings from '../../r2mm/manager/ManagerSettings';
 import ModListSort from '../../r2mm/mods/ModListSort';
 import ProfileModList from '../../r2mm/mods/ProfileModList';
 import SearchUtils from '../../utils/SearchUtils';
-import GameManager from "../../model/game/GameManager";
```

Comment:
I think this whole first commit is leftover from the rebasing and shouldn't exist on this PR anymore (i.e. if this change is actually neede, it should be either on the earlier PR or the main commit of this PR).